packages/core/src/services/commits/create.ts

```diff
@@ -8,19 +8,23 @@ import {
 } from '@latitude-data/core'
 
 export async function createCommit({
-  commit,
+  commit: { projectId, title, mergedAt },
   db = database,
 }: {
-  commit: Omit<Partial<Commit>, 'id'>
+  commit: {
+    projectId: number
+    title?: string
+    mergedAt?: Date
```

Comment:
I prefer to be explicit about what means creating an object. This way people using don't include not permitted columns.

Reply:
👌